lib/topology_docker/nodes/p4switch.py

```diff
@@ -72,50 +72,56 @@ def __init__(
         self.shared_dir = shared_dir
 
         # Add bash shell
-        self._shells['bash'] = DockerShell(
-            self.container_id, 'sh -c "TERM=dumb bash"', 'root@.*:.*# '
-        )
-
-        # Add SAI shell (https://github.com/p4lang/switchsai)
-        self._shells['sai'] = DockerShell(
-            self.container_id,
-            'sh -c "TERM=dumb bash"',
-            'root@.*:.*# ',
-            '/p4factory/targets/switch/tests/pd_thrift/switch_sai_rpc-remote '
-            '-h localhost:9092 '
-        )
-
-        # Add switchapi shell (https://github.com/p4lang/switchapi)
-        self._shells['api'] = DockerShell(
-            self.container_id,
-            'sh -c "TERM=dumb bash"',
-            'root@.*:.*# ',
-            '/p4factory/targets/switch/tests/pd_thrift/switch_api_rpc-remote '
-            '-h localhost:9091 '
+        self._shells['bash'] = DockerBashShell(
+            self.container_id, 'bash'
         )
 
+        # NOTE: It seems that nobody is using this shells.
+        # They got commented when migrating to the new Shell API as not enough
+        # knowledge was gathered to determine how to migrate them.
+
+        # Add SAI shell
+        # https://github.com/p4lang/switch/tree/master/switchsai
+        # self._shells['sai'] = DockerShell(
+        #     self.container_id,
+        #     'sh -c "TERM=dumb bash"',
+        #     'root@.*:.*# ',
+        #     '/p4factory/targets/switch/tests/pd_thrift/switch_sai_rpc-remote'
+        #     ' -h localhost:9092 '
+        # )
+        #
+        # Add switchapi shell
+        # https://github.com/p4lang/switch/tree/master/switchapi
+        # self._shells['api'] = DockerShell(
+        #     self.container_id,
+        #     'sh -c "TERM=dumb bash"',
+        #     'root@.*:.*# ',
+        #     '/p4factory/targets/switch/tests/pd_thrift/switch_api_rpc-remote'
+        #     ' -h localhost:9091 '
+        # )
+        #
```

Comment:
We will leave this here for the mean time.

This comments MUST be removed after nodes migration